src/plcg/structure_scoring/rosetta/rosetta_scores.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+import os
+import pandas as pd
+
+
+def calculate_rosetta_scores(sequence: str, pdb, working_dir: str):
+    pdb_filepath = "workingpdb.pdb"
+    if os.path.exists(working_dir + pdb_filepath):
+        os.remove(working_dir + pdb_filepath)
+    with open(working_dir + pdb_filepath, "wb") as output_file:
+        output_file.write(bytes(pdb[0]))
+    return run_rosetta(sequence, working_dir + pdb_filepath)
+
+
+def run_rosetta(sequence: str, pdb_path: str):
+    out = subprocess.run(
+        [
+            "/mnt/d/case-sensitive/miniconda3/envs/RoseTTAFoldcp35/bin/python",
```

Comment:
If this code is going to be used outside of the workstation, the python interpreter shouldn't be hardcoded